index.html

```diff
@@ -0,0 +1,118 @@
+            tecnológico, isolamento social e pandemia muitas dúvidas foram
+            geradas no coração de jovens que querem decidir sua carreira e muito
+            optaram, e estimulamos aqueles que querem fazer, pelo
+            desenvolvimento de sistemas/tecnologia da informação. Ao mesmo tempo
+            existem poucas ferramentas interativas que possibilitem o
+            entendimento e catequese da palavra de forma didática e ordenada.
+            Por isso esse projeto nasce da necessidade de incentivar aqueles que
+            querem aprender a programar dentro da Igreja, com projetos que serão
+            funcionais e úteis.
+          </p>
+        </aside>
+        <hr />
+        <aside>
+          <p>
+            <span>Criamos</span> documentos para direcionamento do aprendizado
+            com metas e links relevantes.
+          </p>
+          <img src="assets/images/Group 5.png" alt="" />
+        </aside>
+        <hr />
+        <aside>
+          <img src="assets/images/Group 2.png" alt="" />
+          <p>
+            <span>Curadoria</span> do crescimento. Acompanhamos o crescimento e
+            desenvolvimento, direcionando conteúdo se necessário e tirando
+            dúvidas.
+          </p>
+        </aside>
+        <hr />
+        <aside>
+          <img src="assets/images/Group 1.png" alt="" />
+          <p>
+            Algumas vezes fazemos isso copiando por
+            <span>Ligação de vídeo/streaming</span>
+          </p>
+        </aside>
+        <hr />
+        <aside>
+          <img src="assets/images/Group 7.png" alt="" />
+          <p>
+            Mas a maioria das vezes é <span>Comentando código,</span> ou
+            revisando-os
+          </p>
+        </aside>
+        <hr />
+        <aside>
+          <p class="justify">
+            <span>Ambiente colaborativo</span><br />Já dizia minha professora,
+            “a dúvida do coleguinha pode ser a sua”, por isso criamos uma
+            ambiente de perguntas e respostas para facilitar o aprendizado e
+            retirada de dúvidas
+          </p>
+          <img src="assets/images/Group 3.png" alt="" />
+        </aside>
+      </section>
+      <section id="donate-content">
+        <img src="assets/images/Group 9.png" alt="" />
+        <aside>
+          <h2><span>Colabore com o projeto</span></h2>
+          <p class="justify">
+            Você curtiu a ideia, quer contrubuir para a eficiencia do projeto,
+            organização ou simplemente já tem um filho ou amigo no projeto e
+            gostaria de ajudar,
+          </p>
+          <h2 id="doacao">FAÇA UMA DOAÇÃO</h2>
+          <div>
+            <a href=""
+              ><img src="assets/images/patreon_logo_icon_147253 2.png" alt="" />
+              <p>Patreon</p></a
+            >
+          </div>
+        </aside>
+      </section>
+      <section id="empty-right-main"></section>
+      <section id="empty-right-donate"></section>
+    </main>
+    <footer>
```

Comment:
![image](https://github.com/fundamentos-dev/site/assets/34348097/af5dff97-4b24-4177-a317-e40d0e3933ab)

Em dispositivos mobile geralmente usamos [8px](https://uxplanet.org/everything-you-should-know-about-8-point-grid-system-in-ux-design-b69cb945b18d) como unidade base e os [`rem`](https://www.w3schools.com/cssref/tryit.php?filename=trycss_unit_rem)  para determinar seus múltiplos 

![image](https://github.com/fundamentos-dev/site/assets/34348097/8bf27588-2805-4aac-bfe6-1cb2b273619b)

Aqui temos 55px de padding, além de não contribuir para uma visualização harmonica dificulta a leitura do texto pois provoca muitas quebras.

Reply:
Eu estava observando as mudanças em https://wilsonpereirajr.github.io/p-gina-da-fundamento-dev/, se vc puder atualizar lá, agradeço. Facilita mais minhas vida. Rs.